### PR TITLE
HiveCPQ Product Configurator (certified connector)

### PR DESCRIPTION
**Pull request by HiveCPQIntegration**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by HiveCPQIntegration** [comment]
Additional context: the responses that were updated are bugfixes. The described responses did NOT match the actual API response and are causing flows to fail. This was discovered when testing our deployed connector in the preview region for certification, we are not yet live with it.

Is this an issue? As I see Swagger and Change Validation check (understandably) failing? Or can we proceed with this change?

**Comment by HiveCPQIntegration** [comment]
Hello, any news on the review or our question? These last bugfixes are delaying progress in the certification.

File: certified-connectors/HiveCPQ - Product configurator/apiDefinition.swagger.json [hunk 1]
```diff
@@ -10290,31 +10288,8 @@
...
-                          "description": "The configurator input value."
-                        }
-                      },
-                      "description": "The input values for the configurator."
-                    },
-                    "description": "Information about the component if it has been configured as a configurable product."
-                  }
-                },
-                "description": "configurable"
+                "type": "boolean",
```
Comment (vmanoharas) [inline certified-connectors/HiveCPQ - Product configurator/apiDefinition.swagger.json:10291]: Hello @HiveCPQIntegration, 

Can you please confirm that "The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed" 

For more details please check Swagger validation and error and confirm here.

**Comment by HiveCPQIntegration** [comment]
Hello @vmanoharas,

as you reviewed us last time, would you be available to do so again? (or check out our question above?). Our contact in the certification activity control is asking us to provide an update.

Best regards

File: certified-connectors/HiveCPQ - Product configurator/apiDefinition.swagger.json (hunk 1 above)
Comment (vmanoharas) [reply certified-connectors/HiveCPQ - Product configurator/apiDefinition.swagger.json:10291]: Hi @HiveCPQIntegration, 
Can you please address my question above?

**Review by vmanoharas — APPROVED** [review]
Hello @HiveCPQIntegration, 

I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**